kuma/payments/urls.py

```diff
@@ -5,10 +5,6 @@
 lang_urlpatterns = [
     path("terms/", views.payment_terms, name="payment_terms"),
     path("thank-you/", views.thank_you, name="thank_you"),
-    path(
-        "recurring/management/",
-        views.recurring_payment_management,
-        name="recurring_payment_management",
-    ),
```

Comment:
I guess this is for me to fix in https://github.com/mdn/kuma/issues/6898 
Let's just not forget to do it as soon as we land this PR.